### DISCUSSION TITLE
CSV to JSON

### DISCUSSION THREAD

**Discussion by sottey** [CorentinTh/it-tools#658] [Ideas]
I love the app! Thank you!

The only thing I see that I wish was there is a CSV to JSON converter. I see the JSON -> CSV converter, was there a reason that CSV -> JSON wasn't added?

Thanks again!

**Comment by sharevb** [comment]
Hi @sottey , if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
